### PR TITLE
amora[dash] :: UI Enhancements

### PR DESCRIPTION
**Pull request by diogommartins**
- deps(dash): Adds `dash-mantine-components^0.10.2` and `dash-extensions^0.1.5`
- feat(dash): `amora.dash.components.side_bar` redesign 
  - From `Div` to `Offcanvas`. Removes custom styles 
  - Adds "Press `⌘` to toggle the menu" 
  - Adds Login
- feat(dash): Adds `amora.dash.components.user_avatar`
- fix(dash): Fixes `amora.dash.components.dependency_dag` layout
- fix(dash): Removes custom css
- feat(dash): `amora.dash.pages.home` redesign       
  - Removes DAG from home 
  - Adds animation to home
- feat(dash): Adds `amora.dash.config.Color` and size `amora.dash.config.Size
- feat(dash): Redesign `amora.dash.pages.questions` - Renames menu item `Questions` -> `Data Questions` - Adds UI where user can select which questions to render. - Questions are loaded on demand, via callback
- fix(questions): Fixes `amora.questions.Question.__eq__` - Adds `amora.questions.Question.uid`
- feat(dash): Adds animation to `amora.dash.pages.dashboards`
- feat(dash): Handles errors in data questions answer rendering
- feat(dash): Redesigns `dash_app.layout`
- feat(dash): Implements toggle menu on `metaKey` keydown
- feat(dash): Implements click on node to show model details on `amora.dash.components.dependency_dag`
- feat(dash): Redesign `amora.dash.pages.models` - Adds a project lineage DAG - Adds animation
- feat(dag): Styles cytoscape nodes according to model characteristics
- feat(dag): Adds `amora.feature_store.feature_view.is_feature_view(model: Model) -> bool`

## Home

Screen Shot 2022-08-08 at 18 34 12

## Authentication

Screen Shot 2022-08-08 at 18 48 00

## Data Models 

Screen Shot 2022-08-08 at 18 48 33

## Data Questions
Screen Shot 2022-08-08 at 18 44 33

### CODE REVIEW

**Comment by thulio** [comment]
Resolvi os conflitos, mas o mypy ta acusando erros: https://github.com/mundipagg/amora-data-build-tool/runs/7768802759?check_suite_focus=true#step:10:17

File: amora/meta_queries.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ def _summarize_columns(model: Model) -> pd.DataFrame:
         is_supported = not isinstance(column.type, (ARRAY, STRUCT))
         is_numeric = isinstance(column.type, (Numeric, Integer, Float))
 
-        if isinstance(model, FeatureViewSourceProtocol):
+        if is_feature_view(model):
```
Comment (diogommartins) [inline amora/meta_queries.py:36]: Voltarei atrás nessa refatoração porque da forma anterior o mypy não reclamava.

Comment (thulio) [reply amora/meta_queries.py]: https://github.com/mundipagg/amora-data-build-tool/runs/7772830542?check_suite_focus=true#step:10:19

**Comment by thulio** [comment]
Agora os testes estão quebrando 😓

**Pull request merged**